.github/workflows/ci.yaml

```
@@ -51,6 +51,9 @@ jobs:
       - name: Checkout Code
         uses: actions/checkout@v4
 
+      - name: Clean
+        run: rm -f backend/backend.cabal; rm -rf backend/.stack-work
```

```suggestion
        run: rm -f backend/backend.cabal backend/.stack-work
```

https://wiki.ubuntuusers.de/rm/

---

This is a draft pr and its purpose is to debug the ci. Obviously, my approach does not work and your suggestion is just cosmetic code change.

If u really want to contribute in a meaningful way, try to figure out where the error originates from instead of rewriting non-working solutions.

Thank you!